packages/esm-patient-forms-app/src/forms/forms-list.component.tsx

```diff
@@ -63,13 +63,15 @@ const FormsList: React.FC<FormsListProps> = ({ completedForms, error, sectionNam
   const tableRows = useMemo(
     () =>
       filteredForms?.map((formData) => {
-        return {
-          id: formData.form.uuid,
-          lastCompleted: formData.lastCompleted ? formatDatetime(formData.lastCompleted) : undefined,
-          formName: formData.form.display ?? formData.form.name,
-          formUuid: formData.form.uuid,
-          encounterUuid: formData?.associatedEncounters[0]?.uuid,
-        };
+        if (formData) {
```

Comment:
Is this necessary? If so, it seems to indicate we're getting `null` or `undefined` entries in the `filteredForms` array and we should fix that.

Reply:
@ibacher it was just a sanitary check , but i have removed it